### PR TITLE
Update VOC mAP evaluator

### PR DESCRIPTION
**Pull request by ivan-lazarevich**

### CODE REVIEW

**Comment by ivan-lazarevich** [comment]
@ravishk007 @shahrukh-dl @ahmed-deeplite for your review guys

**Comment by ivan-lazarevich** [comment]
mAP before:
{'aeroplane': 0.7144218, 'bicycle': 0.6998919, 'bird': 0.44907528, 'boat': 0.4352311, 'bottle': 0.43243682, 'bus': 0.6787119, 'car': 0.753523, 'cat': 0.5954439, 'chair': 0.43239558, 'cow': 0.4940306, 'diningtable': 0.48109543, 'dog': 0.53291655, 'horse': 0.7106102, 'motorbike': 0.71280164, 'person': 0.72715956, 'pottedplant': 0.29400215, 'sheep': 0.46203667, 'sofa': 0.5056518, 'train': 0.7835761, 'tvmonitor': 0.6361934, 'mAP': 0.576560266315937}

mAP now:
 {'mAP': 0.5765602659611475, 0: 0.7144218, 1: 0.6998919, 2: 0.44907528, 3: 0.4352311, 4: 0.43243682, 5: 0.6787119, 6: 0.753523, 7: 0.5954439, 8: 0.43239558, 9: 0.4940306, 10: 0.48109543, 11: 0.53291655, 12: 0.7106102, 13: 0.71280164, 14: 0.72715956, 15: 0.29400215, 16: 0.46203667, 17: 0.5056518, 18: 0.7835761, 19: 0.6361934}

(numbers are class names in alphabetical order)

File: deeplite_torch_zoo/src/objectdetection/eval/evaluator.py [hunk 1]
```diff
@@ -13,16 +13,13 @@
...
 
 class Evaluator(object):
-    def __init__(self, model, data_path=None, img_size=448, net="yolov3"):
-
-        Path(data_path).mkdir(parents=True, exist_ok=True)
-        self.pred_result_path = data_path
-        self.conf_thresh = hyp_cfg.TEST["conf_thresh"]
-        self.nms_thresh = hyp_cfg.TEST["nms_thresh"]
+    def __init__(self, model, img_size=448,
+        conf_thresh=hyp_cfg.TEST["conf_thresh"], nms_thresh=hyp_cfg.TEST["nms_thresh"]):
```
Comment (ahmed-deeplite) [inline deeplite_torch_zoo/src/objectdetection/eval/evaluator.py:17]: I think it is time to get ride off the hyp config files. We can use default values and send it as param to Evaluator. What do you think?

**Review by shahrukh-dl — APPROVED** [review]
Looks good to me

**Review by ravishk007 — APPROVED** [review]

File: deeplite_torch_zoo/src/objectdetection/eval/evaluator.py (hunk 1 above)
Comment (ivan-lazarevich) [reply deeplite_torch_zoo/src/objectdetection/eval/evaluator.py:17]: Agreed, will make it a part of a next PR

**Pull request merged**